scarlet/component.py

```diff
@@ -90,8 +90,8 @@ def model_to_frame(self, frame=None, model=None):
             Image of the model to project.
             This must be the same shape as `self.bbox`.
             If `model` is `None` then `self.get_model()` is used.
-        frame: `~scarlet.frame.Frame`
-            The frame to project the model into.
+        bbox: `~scarlet.bbox.Box` or `~scarlet.frame.Frame`
```

Comment:
I have one qualm with that (which isn't really changed since we introduced this method): I don't think this method can every accept an actual `Frame` since there it cannot account for resolution/subpixel-position changes. As an option it is sort of disguised now, but code still allows it. I'd feel happier if it really only accepted `Box` here. Is there a case where another `Frame` is needed?

Reply:
No, that's fine. I just wasn't sure how much this was used by either you or @herjy in your own scripts, and I didn't want to mess with the API if I didn't have to. But if you two are fine with it I'm happy to use a box exclusively.